CITATION.cff

```diff
@@ -18,14 +18,14 @@ authors:
   given-names: "Hilmar"
   orcid: "https://orcid.org/0000-0001-9107-0714"
 cff-version: 1.2.0
-date-released: "2024-03-08"
+date-released: "2024-05-17"
```

Comment:
Also update on release

Reply:
Yeah, I was going to change that once this got approved and squash it.